api/src/migrations/add-contact-icons-to-resources-doc.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
             doc.resources[resource.name] = resource.file;
             doc._attachments[resource.file] = {
               content_type: resource.type,
-              data: new Buffer(data).toString('base64')
+              data: Buffer.from(data, 'base64')
```

Comment:
You've changed this implementation now, so instead of returning a String it returns a Buffer. Please change it to:

```js
data: Buffer.from(data).toString('base64')
```